aldryn_jobs/views.py

```diff
@@ -6,12 +6,13 @@
 from django.http import Http404
 from django.shortcuts import redirect
 from django.utils.translation import (
-    ugettext_lazy as _, get_language_from_request
+    ugettext as _, get_language_from_request
```

Comment:
You're probably right about the 90% of the cases, '_' is a shortcut for ugettext_lazy, when it SHOULD be more like 60% of the time. Unfortunately, people tend to be lazy about using the correct method, unfortunately.

Regardless, I'm not sure I agree about '_' should only ever be used for the lazy variant. The fact is, it is painful to type out ugettext* anytime you write a translatable string, this is why they adopted the '_' in the first place. To most Django developers (that I know) _(...) means "here lies a translated string", not "here lies a translated and lazy string".